src/applications/Job.java

```diff
@@ -63,16 +63,43 @@ boolean moveToNextMachine(MachineShopSimulator machineShopSimulator, SimulationR
 	        return false;
 	    } else {// theJob has a next task
 	            // get machine for next task
-	        int p = ((Task) getTaskQ().getFrontElement()).getMachine();
+	        int index = ((Task) getTaskQ().getFrontElement()).getMachine();
 	        // put on machine p's wait queue
-	        machineShopSimulator.machine[p].getJobQ().put(this);
+	        machineShopSimulator.machine[index].getJobQ().put(this);
 	        setArrivalTime(machineShopSimulator.timeNow);
 	        // if p idle, schedule immediately
-	        if (machineShopSimulator.eList.nextEventTime(p) == machineShopSimulator.largeTime) {// machine is idle
-	            machineShopSimulator.changeState(p);
-	        }
+	        ChangeState(machineShopSimulator, index);
 	        return true;
 	    }
 	}
 
+	private void ChangeState(MachineShopSimulator machineShopSimulator, int index) {
```

Comment:
The Java "standard" (not enforced by the language) is that method names start with lowercase letters, and classes start with uppercase letters.

```suggestion
	private void changeState(MachineShopSimulator machineShopSimulator, int index) {
```

That's getting close to arguments about indentation, but you'd rather not use uppercase like this in a job interview.

Reply:
I meant this to be a just the  extraction of the methods in a class and then to remove that class but then a different pull request got merged in to main that we should have also assigned reviewers to. Sorry this will not happen again a bit of miscommunication by our group.

Reply:
The capital 'C' in `changeState` was more of a mistake I made with working on the code... long story short I had to remake the method and I think i made the C capital by mistake.